src/pages/billing/List.js

```diff
@@ -42,7 +43,47 @@ const styles = {
   }
 }
 
-const ProjectListItem = ({ project, project: { roles, status }, isActive }) => {
+const BillingProjectActions = ({ project: { projectName }, loadProjects }) => {
+  const signal = useCancellation()
```

Comment:
I disagree with using cancellation here. If the user triggers a delete operation and then leaves the page, I think they would still expect the delete to happen. We typically use cancellation for GET calls where we don't want to finish them if the user leaves the page.

Looking at other DELETE ajax calls, we do not use cancellation for those.